content/docs/faq-styling.md

```diff
@@ -40,7 +46,7 @@ CSS classes are generally more efficient than inline styles.
 
 ### What is CSS-in-JS?
 
-CSS-in-JS refers to a pattern where CSS is written with Javascript, then extracted into a stylesheet.
+CSS-in-JS refers to a pattern where CSS is written with Javascript, then extracted into a stylesheet.It is `not mandatory` to use CSS-in-JS.
```

Comment:
Nit: Missing a spacer in between "stylesheet.It" Also the use of back-ticks for emphasis doesn't fit with our formatting elsewhere.

This sentence feels a bit incomplete. What do you think about expanding it slightly?

> CSS-in-JS refers to a pattern where CSS is written with Javascript, then extracted into a stylesheet.

> It is _not mandatory_ to use CSS-in-JS. You may define your styles in separate `*.css` files or inline (using the `style` attribute) if you prefer.

Reply:
Even "it is not mandatory" sounds too light to me, as if it implies we still recommend it (but don't force people to use it). Whereas I don't even think we recommend it.

Reply:
Yes I too feel that we need to expand it slightly. Let me come up with newer version.@gaearon @bvaughn

Reply:
Good point Dan!

I like the wording that you ended up with, @jyash97. Great iteration.